### PR TITLE
Feature skip_preload

### PR DESCRIPTION
**Pull request by OuYangJinTing**
Thank for this gem and all contributors. This gem is great, auto-preload config is convenient. 
But sometimes we don’t need to loads all records association auto matically, just load AR own associations.
This PR is trying to deal with this scene.

### CODE REVIEW

**Comment by PikachuEXE** [comment]
1. I don't quite understand what this does
Imagine you are writing README section about this new method, what would you write?

2. Commit `Fix sql matching` seems strange as the previous commit got a pass in CI build
Or is it just a refactor (in spec)?

**Comment by OuYangJinTing** [comment]
> 1. I don't quite understand what this does
>    Imagine you are writing README section about this new method, what would you write?
> 2. Commit `Fix sql matching` seems strange as the previous commit got a pass in CI build
>    Or is it just a refactor (in spec)?

The `#skip_preload` is  for avoid preload all associations at special scene.
I try to udpate README, `#skip_preload` add to `Auto preloading` section.
About `Fix sql matching` commit, some rails version CI build is failed, it has two different SQL cases.
```sql
SELECT "posts".* FROM "posts" WHERE "posts"."user_id" IN (1, 2)
SELECT "posts".* FROM "posts" WHERE "posts"."user_id" IN (?, ?)  [["user_id", 1], [user_id", 2]]
```

**Comment by PikachuEXE** [comment]
I think your README update is incomplete/broken...

**Comment by OuYangJinTing** [comment]
> I think your README update is incomplete/broken...

Very much your remind. I'm so sorry, ignored `skip_preload` method call.

File: lib/ar_lazy_preload/active_record/collection_association.rb [hunk 1]
```diff
@@ -16,6 +16,6 @@ def ids_reader
         @association_ids ||= reader.map(&primary_key)
       end
     end
-    # rubocop:enable Metrics/AbcSize
+    # rubocop:enable Metrics/AbcSize, Metrics/CyclomaticComplexity
```
Comment (DmitryTsepelev) [inline lib/ar_lazy_preload/active_record/collection_association.rb:19]: Looks like we do not need to disable `Metrics/CyclomaticComplexity` here

File: README.md [hunk 2]
```diff
@@ -44,6 +44,11 @@ ArLazyPreload.config.auto_preload = true
 
 After that there is no need to call `#lazy_preload` on the association, everything would be loaded lazily.
 
+If you don't want to load all records's associations, you can use `skip_preload` before associations method, it only load single record's associations:
```
Comment (DmitryTsepelev) [inline README.md:47]:
```suggestion
If you want to turn automatic preload off for a specific record, you can call `.skip_preload` before any associations method:
```

File: lib/ar_lazy_preload/active_record/collection_association.rb (hunk 1 above)
Comment (OuYangJinTing) [reply lib/ar_lazy_preload/active_record/collection_association.rb:19]: image
Remove `Metrics/CyclomaticComplexity`, rubocop will discover an offense.
You can see https://github.com/rubocop-hq/rubocop/pull/8149

File: README.md (hunk 2 above)
Comment (OuYangJinTing) [reply README.md]: Thank you for your correction!

**Pull request merged**